e2e/README.md

```diff
@@ -23,7 +23,7 @@ This flow enables deploying and running 2 applications locally.
 1. From the e2e directory, run
 
     ```sh
-    pip install -r requirements.txt
+    pip install -r requirements-ray.txt
```

Comment:
I think that we need to pip install both.

Reply:
Wait. `requirements-ray.txt` is self sufficient for inference. We don't need to pip install both.